src/CommentFooter.tsx

```diff
@@ -0,0 +1,143 @@
+import * as React from 'react';
+
+interface ICommentFooter {
+  expanded?: boolean;
+  handleExpand?: any;
+  replyActive?: boolean;
+  handleReplyActive?: any;
```

Comment:
Could we add type signatures to these? Are they functions? https://www.typescriptlang.org/docs/handbook/functions.html#writing-the-function-type

Reply:
I added the signature "VoidFunction" and it works still, is that type correct?